include/pros/apix.h

```diff
@@ -555,4 +556,107 @@ int32_t fdctl(int file, const uint32_t action, void* const extra_arg);
 }
 #endif
 
+#ifdef __cplusplus
+namespace pros {
+extern "C" {
```

Comment:
is there a reason these can't be the same block? these functions should be in the `pros::c` namespace anyway

Reply:
Make sense

Reply:
Updated, and now we have a messy namespace. Look at the test code.